views/layouts/layout-pane.njk

```diff
@@ -1,22 +1,30 @@
 <!doctype html>
-<html>
+<html class="no-js">
   <head>
     <meta charset="utf-8">
     <meta name="robots" content="noindex, nofollow">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{ title }}</title>
     <link href="/stylesheets/main.css" rel="stylesheet" media="all" />
-    <!--[if lt IE 9]><script src="/javascripts/ie.js"></script><![endif]-->
-    <script src="/javascripts/application.js"></script>
+    <!--[if lt IE 9]>
+      <script src="/javascripts/ie.js"></script>
+    <![endif]-->
+    <script src="/javascripts/vendor/jquery-1.12.0.js"></script>
```

Comment:
Should this be included as part of application.js in the build pipeline?

Reply:
/cc @alex-ju

Reply:
it is https://github.com/alphagov/govuk-design-system/pull/83/files#diff-3ce979ee0fbe6a5be1d682b0299ee40fR54

Reply:
Yes I'm aware that jQuery is also part of #83 👍